src/controllers/controller.line.js

```diff
@@ -6,8 +6,9 @@ import {_lookupByKey} from '../helpers/helpers.collection';
 export default class LineController extends DatasetController {
 
   initialize() {
-    this.enableOptionSharing = true;
     super.initialize();
+    this.enableOptionSharing = true;
```

Comment:
I think I originally placed this before `initialize` on purpose, but maybe the initialization code has changed after that. Can't find a reason for it anymore.

Reply:
Will move it back since we'll initialize to `false` in the constructor